lib/Sapi.php

```diff
@@ -67,7 +67,20 @@ static function sendResponse(ResponseInterface $response) {
             }
 
         }
-        file_put_contents('php://output', $response->getBody());
+
+        if(is_null($response->getBody())) return;
+
+        if($response->hasHeader('Content-Length')) {
```

Comment:
A faster way is to replace `hasHeader` + `getHeader` by `null !== $contentLength = $response->getHeader(…)`.